__test__/e2e/page-functions/campaigns.js

```diff
@@ -0,0 +1,124 @@
+const { until } = require('selenium-webdriver')
+
+const remote = require('selenium-webdriver/remote')
+import { wait } from '../util/helpers'
+const pom = {}
+pom.campaign = require('../page-objects/campaigns')
+pom.navigation = require('../page-objects/navigation')
+
+module.exports = {
+  startCampaign(driver, campaign) {
+    it('opens the Campaigns tab', async () => {
+      await wait.andClick(driver, pom.navigation.sections.campaigns)
+    })
+
+    it('clicks the + button to add a new campaign', async () => {
+      await wait.andClick(driver, pom.campaign.add)
+    })
+
+    it('completes the Basics section', async () => {
+      // Title
+      await wait.andType(driver, pom.campaign.form.basics.title, campaign.basics.title)
+      // Description
+      await wait.andType(driver, pom.campaign.form.basics.description, campaign.basics.description)
+      // Select a Due Date using the Date Picker
+      /**
+       * Date Picker Notes:
+       * The selector for the date is fragile. It may be better to programatically set it.
+       * await driver.executeScript('document.getElementsByName("dueBy")[0].setAttribute("value","10 Jan 2019")')
+       * Similarly, a sleep is added because it's difficult to know when the picker dialog is gone.
+       */
+      await wait.andClick(driver, pom.campaign.form.basics.dueBy)
+      await wait.andClick(driver, pom.campaign.form.datePickerDialog.nextMonth)
+      await driver.sleep(1000) // Transition
+      await wait.andClick(driver, pom.campaign.form.datePickerDialog.enabledDate)
+      await driver.sleep(3000)
+      // Save
+      await wait.andClick(driver, pom.campaign.form.save)
+      // This should switch to the Contacts section
+    })
+
+    it('completes the Contacts section', async () => {
+      await driver.setFileDetector(new remote.FileDetector()) // TODO: maybe this belongs earlier?
+      const el = await driver.wait(until.elementLocated(pom.campaign.form.contacts.input), 10000)
+      await el.sendKeys(campaign.contacts.csv)
+      await wait.andGetEl(driver, pom.campaign.form.contacts.uploadedContacts)
+      expect(await wait.andGetEl(driver, pom.campaign.form.contacts.uploadedContacts)).toBeDefined()
+      // Save
+      await wait.andClick(driver, pom.campaign.form.save)
+      // This should switch to the Texters section
```

Comment:
can we confirm the switch by looking for another element on the texters section? (maybe that should be a pattern for each section)